Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionProgressReporterListener.cs

```diff
@@ -134,21 +134,34 @@ public static void AddExecutionDetailsToLog(eExecutionPhase objExecutionPhase, s
                                 {
                                     string propName = prop.Name;
                                     string propFullName = ((FieldParamsNameCaption)prop.GetCustomAttribute(typeof(FieldParamsNameCaption))).NameCaption;
-                                    object propVal = prop.GetValue(obj);//obj.GetType().GetProperty(propName, BindingFlags.Public | BindingFlags.Instance).GetValue(obj).ToString();
-                                    string propValue;
-                                    if (propVal != null)
+                                    object propValueobj = prop.GetValue(obj);//obj.GetType().GetProperty(propName, BindingFlags.Public | BindingFlags.Instance).GetValue(obj).ToString();
+                                    string propValueStr = "";
+                                    if (propValueobj != null)
                                     {
-                                        propValue = propVal.ToString();
+                                        //special case for execution time convertion from UTC format
+                                        if (Attribute.IsDefined(prop, typeof(UsingUTCTimeFormat)))
+                                        {
+                                            try
+                                            {
+                                                propValueStr = ((DateTime)propValueobj).ToLocalTime().ToString("dd-MMM-yy HH:mm:ss");
+                                            }
+                                            catch
+                                            {
+                                                propValueStr = propValueobj.ToString();
+                                            }
+                                        }
+                                        else
+                                        {
+                                            propValueStr = propValueobj.ToString();
+                                        }
                                     }
-                                    else
-                                    {
-                                        propValue = "";
-                                    }
-
-                                    stringBuilder.Append(propFullName).Append("= ").Append(propValue).AppendLine();
+                                    
+                                    stringBuilder.Append(propFullName).Append("= ").Append(propValueStr).AppendLine();
                                 }
                             }
-                            catch (Exception) { }
+                            catch (Exception ex) 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)